proto/authzed/api/validation_test/relationships_test.go

```diff
@@ -84,6 +84,7 @@ var objectIDs = []struct {
 	{"authn|", true},
 	{"authn|a-b-c-d-e", true},
 	{"authn|a_b", true},
+	{"--=base64YWZzZGZh-ZHNmZHPwn5iK8J+YivC/fmIrwn5iK==", true},
```

Comment:
What about the very long object ID case?

Reply:
https://github.com/authzed/authzed-go/pull/102/files#diff-a771dac00d07460a29966864dccdd10ecc0108ac295bb0bcaa32d21401ff5a75R70